PowerEditor/src/Notepad_plus.h

```diff
@@ -563,6 +563,7 @@ friend class FileManager;
 	void autoCompFromCurrentFile(bool autoInsert = true);
 	void showFunctionComp();
 	void showPathCompletion();
+	void showFunctionNext();

```

Comment:
Since we have added new item "Function Parameters Previous Hint", the function prototype should be:
`void showFunctionNextHint(bool isNext); // set isNext to false to reach the previous one`